wagtail/wagtaildocs/tests.py

```diff
@@ -576,3 +578,39 @@ def test_with_nonexistent_document(self):
     def test_with_incorrect_filename(self):
         response = self.client.get(reverse('wagtaildocs_serve', args=(self.document.id, 'incorrectfilename')))
         self.assertEqual(response.status_code, 404)
+
+
+class TestDocumentLinkHandler(TestCase):
```

Comment:
Could we have a little more context in this name? ``TestDocumentRichTextLinkHandler``